.github/workflows/ci.yml

```diff
@@ -0,0 +1,119 @@
+            ~/.cache/coursier/v1
+            ~/AppData/Local/Coursier/Cache/v1
+            ~/Library/Caches/Coursier/v1
+          key: ${{ runner.os }}-sbt-cache-v2-${{ hashFiles('**/*.sbt') }}-${{ hashFiles('project/build.properties') }}
+
+      - name: Check that workflows are up to date
+        run: sbt ++${{ matrix.scala }} githubWorkflowCheck
+
+      - name: Build project
+        run: sbt ++${{ matrix.scala }} test
+
+      - name: Compress target directories
+        run: tar cf targets.tar joda/target/1-16-jvm-2.13 joda/target/1-16-jvm-3 core/target/1-13-jvm-2.11 core/target/1-15-jvm-2.13 joda/target/1-13-jvm-2.11 joda/target/1-13-jvm-2.12 core/target/1-13-jvm-2.12 core/target/1-15-jvm-2.12 joda/target/1-12-jvm-2.11 joda/target/1-15-jvm-3 joda/target/1-15-jvm-2.13 core/target/1-16-jvm-3 joda/target/1-14-jvm-2.13 target core/target/1-15-jvm-3 joda/target/1-14-jvm-2.12 core/target/1-14-jvm-2.11 core/target/1-16-jvm-2.13 joda/target/1-16-jvm-2.12 joda/target/1-15-jvm-2.12 core/target/1-16-jvm-2.12 core/target/1-14-jvm-2.12 joda/target/1-14-jvm-2.11 core/target/1-12-jvm-2.11 core/target/1-14-jvm-2.13 project/target
+
+      - name: Upload target directories
+        uses: actions/upload-artifact@v2
+        with:
+          name: target-${{ matrix.os }}-${{ matrix.scala }}-${{ matrix.java }}
+          path: targets.tar
+
+  publish:
+    name: Publish Artifacts
+    needs: [build]
+    if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/v'))
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        scala: [2.12.15]
+        java: [temurin@11]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout current branch (full)
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Java (temurin@11)
+        if: matrix.java == 'temurin@11'
+        uses: actions/setup-java@v2
+        with:
+          distribution: temurin
+          java-version: 11
+
+      - name: Cache sbt
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.sbt
+            ~/.ivy2/cache
+            ~/.coursier/cache/v1
+            ~/.cache/coursier/v1
+            ~/AppData/Local/Coursier/Cache/v1
+            ~/Library/Caches/Coursier/v1
+          key: ${{ runner.os }}-sbt-cache-v2-${{ hashFiles('**/*.sbt') }}-${{ hashFiles('project/build.properties') }}
+
+      - name: Download target directories (2.12.15)
+        uses: actions/download-artifact@v2
+        with:
+          name: target-${{ matrix.os }}-2.12.15-${{ matrix.java }}
+
+      - name: Inflate target directories (2.12.15)
+        run: |
+          tar xf targets.tar
+          rm targets.tar
+
+      - env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+        run: sbt ++${{ matrix.scala }} ci-release
```

Comment:
There is an issue with sbt returning an error even though the artifacts are published... I suspected we will need to have some kind of custom code for handling this in the future (or find / fix the bug). But for now, and automated workflow that fails the same way it does locally, but also uploads the artifacts seems preferable to not having any workflow for this.

Reply:
Is it possible to this as a comment in `WorkflowStep.Sbt(List("ci-release"))` so that people are aware of this problem, ideally with a link to a public ticket?